### PR TITLE
chore: use teams instead of individuals in CODEOWNERS

### LINKED ISSUE #199: our CODEOWNERS file has issues
I've kept wondering why global owners' reviews aren't counted as binding for doc changes...

Now I realize this is because the rule for `/docs/` _overrides_ the global rule rather than just _adding_ reviewers for that one dir.

Easy to fix.

### PR DESCRIPTION
**Pull request by alexmt**
Swapping individuals with teams in CODEOWNERS to avoid spamming everyone with emails on each PR.

Also closes #199

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This file is described here: https://help.github.com/en/articles/about-code-owners
 
 # Global owners
-* @alexmt @jessesuen @krancour @rbreeze @devholic @kencochrane @shelby-moore @evgeny-goldin
+* @akuity/maintainers
 
 # Doc owners
-/docs/ @morey-tech @dhpup
+/docs/ @akuity/doc-maintainers
```
Comment (krancour) [inline CODEOWNERS:7]:
```suggestion
/docs/ @akuity/maintainers @akuity/doc-maintainers
```

If you do this, it will also close #199

Comment (alexmt) [reply CODEOWNERS]: agree!

**Review by krancour — APPROVED** [review]

**Comment by krancour** [comment]
@alexmt before you merge... GitHub says this file contains errors... not sure why.

**Comment by alexmt** [comment]
Thanks for the heads up. Github was warning that teams don't have merge permissions. Fixed it.

**Pull request merged**